metaflow/plugins/kfp/README.md

```diff
@@ -26,10 +32,16 @@ You can export the required variables* individually and run the python command:
     "METAFLOW_AWS_ARN" : "...", # required to be able to read s3 data when running within KFP
     "METAFLOW_AWS_S3_REGION" : "...", # specify s3 region being used
 
-    "KFP_RUN_URL_PREFIX" : "https://kubeflow.corp.zillow-analytics-dev.zg-int.net/pipeline/#/runs/details" # prefix of the URL preceeding the run-id to generate correct links to the generated runs on your cluster
+    "KFP_RUN_URL_PREFIX" : "https://kubeflow.corp.dev.zg-aip.net" # prefix of the URL preceeding the run-id to generate correct links to the generated runs on your cluster
+
+    "KFP_SDK_NAMESPACE": "rent-zestimate"
+    "KFP_SDK_USERID": "[YOUR_NAME]@email.com"
 }
 ```
 
+NOTE: make sure you have access to the `rent-zestimate` namespace. Please talk to @Sharon Lyu to obtain access if needed. Also, if you provide either the
```

Comment:
Sharon? Please change all this to be not Zillow specific

Reply:
Done.